### PR TITLE
[#116373897] Add edit and update actions for Projects

### PR DESCRIPTION
**Pull request by drench**
This adds a basic interface for editing projects by adding an edit button to the project show view.

It also includes a few `active` flag to the `Project` model. A checkbox appears on the edit form to toggle `active` but the flag has no impact on anything else yet.

### CODE REVIEW

File: vendor/engines/projects/app/views/projects/projects/edit.html.haml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+= render "shared/headers/ckeditor"
+
+= content_for :h1 do
+  = @project.facility
+
+%h2= t(".update")
+
+= simple_form_for @project, url: facility_project_path(@project.facility, @project), method: :put do |f|
```
Comment (jhanggi) [inline vendor/engines/projects/app/views/projects/projects/edit.html.haml]: can we share the form with `new`?

Comment (drench) [inline vendor/engines/projects/app/views/projects/projects/edit.html.haml]: It's close; `edit` has the `:active` checkbox and `new` doesn't. I lean toward keeping them separate for now.

Comment (jhanggi) [inline vendor/engines/projects/app/views/projects/projects/edit.html.haml]: Ok, that's fine.

**Comment by jhanggi** [comment]
👍

**Pull request merged**